### ISSUE TITLE
Two way bind with ngModel and ngPackagr

### ISSUE THREAD

**Issue by mariohmol** [ng-packagr/ng-packagr#1151]
# Bug Report

## Description

I have a angular component with ng-packagr and this works good. But now i'm trying to make a component that receives an `[(ngModel)]` property.

It works very good locally, but when i compile this with ng-packagr and include in a project, i receive an error:

```
 1. If 'json-editor' is an Angular component and it has 'ngModel' input, then verify that it is part of this module.
        2. If 'json-editor' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
        3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("
        <div style="height:600px;">
          <json-editor [options]="editorOptions" (change)="changeLog($event)"  [ERROR ->][(ngModel)]="data"  #editorTwo ></json-editor>
        </div>
```

#### How To Reproduce

This is an open source project, so its easy to check out

* https://github.com/mariohmol/ang-jsoneditor

Here is the building breaking:

https://travis-ci.org/mariohmol/ang-jsoneditor

#### Expected Behaviour

To execute the ngModel. It seems that some info about the NgModule has not been exported:

```
schemas: [
    CUSTOM_ELEMENTS_SCHEMA,
    NO_ERRORS_SCHEMA
  ]
```

#### Version Information

```
ng-packagr:            3.0.3
@angular/compiler:     7.1.0
rollup:                0.59.4
tsickle:               0.29.0
typescript:            3.1.6
```

**Comment by macjohnny** [comment]
I think you are missing the `ReactiveFormsModule` import in your module

**Comment by ashokv7s** [comment]
Hi,

this is issue is similar to using ngmodel, ngif, ngclass etc directives of angular. Remember all work because of **CommonModule** imported in a regular angular application in app.module.

So Similarly, import it in your library's root module.

for reference see https://github.com/Napster2210/ngx-spinner/blob/master/projects/ngx-spinner/src/lib/ngx-spinner.module.ts of https://www.npmjs.com/package/ngx-spinner
hosted at https://github.com/Napster2210/ngx-spinner#readme.

Do let me know if it worked?

**Comment by mariohmol** [comment]
Hi, thanks for getting in touch!

I have this already, no?

https://github.com/mariohmol/ang-jsoneditor/blob/master/ang-jsoneditor/src/lib.module.ts#L8

**Comment by mariohmol** [comment]
This was missing in the test project 
https://github.com/mariohmol/ang-jsoneditor/blob/master/src/app/app.module.ts#L21

But had the same issue https://travis-ci.org/mariohmol/ang-jsoneditor/builds/463849187

**Comment by qballer** [comment]
hi @mariohmol did you ever resolve this issue?

**Comment by mariohmol** [comment]
no.. i could make the formControl to work.. but not the ngModel
